tests/test_relative_free_energy.py

```diff
@@ -281,6 +285,48 @@ def u(x):
     assert gaussian_overlap((0, 0.1), (0.5, 0.2)) > 0.1
 
 
+@given(integers(min_value=1))
+@seed(2023)
+def test_batches_of_nothing(batch_size):
+    assert list(batches(0, batch_size)) == []
+
+
+@given(integers(min_value=1, max_value=1000), integers(min_value=1))
+@seed(2023)
+def test_batches(n, batch_size):
+    assert sum(batches(n, batch_size)) == n
+    assert all(batch == batch_size for batch in list(batches(n, batch_size))[:-1])
+    *_, last = batches(n, batch_size)
+    assert 0 < last <= batch_size
+
+
+@pytest.fixture(scope="module")
+def hif2a_ligand_pair_single_topology_lam0_state():
+    mol_a, mol_b, core = get_hif2a_ligand_pair_single_topology()
+    forcefield = Forcefield.load_from_file(DEFAULT_FF)
+    st = SingleTopology(mol_a, mol_b, core, forcefield)
+    [state] = setup_initial_states(st, None, DEFAULT_TEMP, [0.0], 2023)
```

Comment:
I find this syntax of setting the variable to a list confusing and hard to read (didn't even know it was possible). Would be nice to make it more explicit.

Reply:
I preferred this method here because it's a little stricter (would fail if the returned list has more than one element), but can use the more familiar method and an extra assertion if preferable

Reply:
Having declaration be a side effect seems confusing and seems unique to `[]` where as `(x) = "foo"` and `(x,) = "foo"` behave differently.

Reply:
Changed in https://github.com/proteneer/timemachine/pull/950/commits/39dbf2e5f9a9210325c0be46995cca7a99c613a9